### ISSUE TITLE
Blackscreen in Gamemode SteamDeck

### ISSUE THREAD

**Issue by KingForKings** [FeralInteractive/gamemode#441]
Can someone make a script or more official feature to reset all display configuration for GameMode so that people do not get stuck with black screen? I have luck I own a VGA CRT too. My LG C1 OLED was or is stuck on Blackscreen and with no safe mode in SteamDeck GameMode, the Deck even crashes completely.

There is not much to say. This can be fixed by owning an other screen and crossing the fingers to have this fix the stuff, so that people can return to the main PC monitor or TV.

How to replicate it? Sadly only bad luck can make it appear. I think it started with beta versions, then I went to Dev-Main and I even had this on Desktop mode which required me to use Dolphin in game mode to delete /home/deck/.local/share/kscreen/ at a time when I knew even less about Linux Arch and Linux in general .

TL;DR I need a fix for BlackScreen on Desktop and Gamemode which will reset all configurations and due to that repair a missing video output. The output is either blackscreen with video or completely no video.

Frequency of this  = low users mentions over 1+ years
Frequency of End User using wrong TV or Monitor settings = little higher
